src/components/common/TmBalance.vue

```diff
@@ -92,6 +148,85 @@ export default {
     },
     onSend() {
       this.$refs.SendModal.open()
+    },
+    async calculateTotalFiatValue() {
```

Comment:
As you are stating in your comments, this is highly tight to specific networks (e-money). But the FE should be stupid and don't know much about individual networks. We need it to be that way so we can scale to many networks. This whole logic needs to be abstracted and put into the API I think.

Reply:
Yes, in fact I considered this. Then I thought, 'OK, but the user needs to select a fiat currency in the FE'... But yes, a simple call to API and that's it.

It is something I definitely thought.